.github/workflows/create_gh_discussion.yml

```
@@ -130,10 +137,14 @@ jobs:
               }
             })();
 
-            const discussionId = await createDiscussion(
+            const createDiscussionResult = await createDiscussion(
               github,
               repositoryIdAndCategoryId.repositoryId,
               repositoryIdAndCategoryId.categoryId,
               discussionTitle,
               description,
             );
+
+            core.setOutput("discussion_id", createDiscussionResult.createDiscussion.discussion.id);
```

(2/2) createDiscussion 関数の使い手が、createDiscussionResult の実装を知らなくて済みます。

```suggestion
            const discussionId = await createDiscussion(
              github,
              repositoryIdAndCategoryId.repositoryId,
              repositoryIdAndCategoryId.categoryId,
              discussionTitle,
              description,
            );

            core.setOutput("discussion_id", discussionId);
```